docs/UserGuide.md

```diff
@@ -94,7 +94,9 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [r/REMARK] [t/TAG]…​`
   * Invalid inputs may include those that contain special characters such as `*`, `@`, `#`, `!`, `^`, `%`, `$`, or other characters that are not English, such as Arabic, Chinese or Latin scripts like `Æ`.
 * The `REMARK` input should be a string of words with a limit of 120 characters.
 
-:exclamation: **Important:** Two identical person with duplicated fields cannot be added into the address book. The duplicated fields refers to `NAME` and `PHONE_NUMBER` fields where combined it should form a unique person.<br>
+<div markdown="span" class="alert alert-danger">
+:exclamation: Important: Duplicate entries for a person cannot be added to the address book. A person is considered a duplicate if both the `NAME` and `PHONE_NUMBER` fields match an existing entry. Together, these fields must create a unique identifier for each person in the address book.
```

Comment:
```suggestion
:exclamation: Important: Duplicate entries for a person cannot be added to the address book. A person is considered a duplicate if both the `NAME` and `PHONE_NUMBER` fields match an existing entry. Together, these fields would create a unique identifier for each person in the address book.
```

Reply:
In this context, “must” is the best choice because it conveys a strong requirement that these fields create a unique identifier to prevent duplicates. It implies that the rule is mandatory for the system to function as intended.

Using “should” would imply a recommendation or best practice rather than a strict rule, which may lead to ambiguity about whether uniqueness is required.

Using “would” can also make the statement sound hypothetical, which may not effectively convey the necessity of uniqueness in this case.

So, “must” is the clearest and most assertive option for establishing this requirement.